### ISSUE TITLE
Debian package

### ISSUE THREAD

**Issue by gjedeer** [gjedeer/tuntox#44]
The work is basically done: https://github.com/abbat/tox.pkg/commit/44f27bc69be059b5d6356e5b9e71fb5aaf2b83e0

**Comment by gjedeer** [comment]
New location: https://github.com/anthonybilinski/tox.pkg/commit/44f27bc69be059b5d6356e5b9e71fb5aaf2b83e0

**Comment by im-grey** [comment]
Thank you.

**Comment by gjedeer** [comment]
Toxcore is in sid https://packages.debian.org/sid/libtoxcore2

**Comment by gjedeer** [comment]
There's now a debian/ subdirectory in the tuntox source: https://github.com/gjedeer/tuntox/tree/master/debian

I'm building deb files using Bitbucket CI here: https://bitbucket.org/gjedeer/tuntox/downloads/

The CI runs: https://bitbucket.org/gjedeer/tuntox/addon/pipelines/home

There's also a Debian section in BUILDING.md

What's left to do is creating a PPA and find a willing maintainer

**Comment by afcady** [comment]
Nice. Debian packages built and appeared to work fine for me, even using Debian `stretch` (with backported `libtoxcore-dev`).

**Comment by gjedeer** [comment]
Yeah, on anything with libtoxcore-dev installed a quick `make` should be enough.
